test/testdata/definition_validator/attr_accessor_override_error.rb

```diff
@@ -0,0 +1,102 @@
+# typed: true
+
+module Fooable
+  extend T::Sig
+  extend T::Helpers
+  interface!
+
+  sig {abstract.returns(T.nilable(String))}
+  def foo; end
+end
+
+# This is incorrect: attr_accessor adds both a reader and a writer
+class BadFooable
+  extend T::Sig
+  include Fooable
+
+  sig {override.returns(T.nilable(String))}
+  attr_accessor :foo # error: Method `BadFooable#foo=` is marked `override` but the parent only defines a writer method
+end
+
+# This is correct: Separate reader and writer
+class GoodFooable
+  extend T::Sig
+  include Fooable
+
+  sig {override.returns(T.nilable(String))}
+  attr_reader :foo
+
+  sig {params(foo: T.nilable(String)).void}
+  attr_writer :foo
+end
+
+module WriterFooable
+  extend T::Sig
+  extend T::Helpers
+  interface!
+
+  sig {abstract.params(value: T.nilable(String)).void}
+  def foo=(value); end
+end
+
+# This is incorrect: attr_accessor adds both a reader and a writer
+class WriterOnlyBad # error: Missing definition for abstract method `WriterFooable#foo=` in `WriterOnlyBad`
+  extend T::Sig
+  include WriterFooable
+
+  sig {override.returns(String)}
+  attr_accessor :bar # error: Method `WriterOnlyBad#bar=` is marked `override` but the parent only defines a writer method
+# ^^^^^^^^^^^^^^^^^^ error: Method `WriterOnlyBad#bar` is marked `override` but the parent only defines a reader method
```

Comment:
This is still wrong. We should report two errors, one saying that `bar` doesn't override anything, another saying `bar=` doesn't override anything, but we should not say that "the parent only defines a reader/writer" in either of those error messages. `bar` is unrelated to `foo`.

The fix for this problem would be to get rid of the `override` keyword in the sig for the `attr_accessor`, because the method is unrelated.